site/docs/guides/chat-scenario.mdx

```diff
@@ -0,0 +1,76 @@
+---
+sidebar_position: 2
+title: Chat Scenario
+---
+
+# Using OpenVINO GenAI in Chat Scenario
+
+For chat applications, OpenVINO GenAI provides special optimizations to maintain conversation context and improve performance using KV-cache.
```

Comment:
Add a link to empty page in the docs describing KV-cache. I guess `HOW_IT_WORKS.md` will be moved there.

Reply:
Link added